packages/squiggle-lang/__tests__/TS/Parser_test.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  run,
+  squiggleExpression,
+  errorValue,
+  result,
+} from "../../src/js/index";
+import { testRun } from "./TestHelpers";
+import * as fc from "fast-check";
+
+describe("Squiggle is whitespace insensitive", () => {
```

Comment:
"The parser is whitespace insensitive" or something more specific

Reply:
will do.